plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/plugins/EditorTabsColorizer.java

```diff
@@ -80,7 +81,12 @@ public void onGitStatusChanged(String endpointId, StatusChangedEventDto statusCh
         .get()
         .getOpenedEditors()
         .stream()
-        .filter(editor -> editor instanceof HasVcsChangeMarkerRender)
+        .filter(
+            editor ->
+                editor instanceof HasVcsChangeMarkerRender
+                    && statusChangedEventDto
+                        .getProjectName()
+                        .equals(getRootProject((File) editor.getEditorInput().getFile()).getName()))
```

Comment:
Check if editor.getEditorInput() instance of File

Reply:
done